.prettierrc.json

```diff
@@ -5,6 +5,9 @@
   "trailingComma": "es5",
   "arrowParens": "avoid",
   "jsxBracketSameLine": false,
-  "plugins": ["prettier-plugin-svelte"],
+  "endOfLine": "auto",
```

Comment:
Why has this been added?

Reply:
this is for windows machines where it flags as a prettier error on every page if not present